test/expected/append-11.out

```diff
@@ -1237,49 +1249,105 @@ ORDER BY time DESC;
+                                                             QUERY PLAN                                                             
+------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (ChunkAppend) on metrics_timestamptz (actual rows=648 loops=1)
+   Order: metrics_timestamptz."time"
+   Chunks excluded during startup: 3
+   ->  Index Only Scan Backward using _hyper_5_17_chunk_metrics_timestamptz_time_idx on _hyper_5_17_chunk (actual rows=336 loops=1)
+         Index Cond: ("time" < ('2000-01-10'::cstring)::timestamp with time zone)
+         Heap Fetches: 336
+   ->  Index Only Scan Backward using _hyper_5_18_chunk_metrics_timestamptz_time_idx on _hyper_5_18_chunk (actual rows=312 loops=1)
+         Index Cond: ("time" < ('2000-01-10'::cstring)::timestamp with time zone)
+         Heap Fetches: 312
+(9 rows)
+
+-- test constraint exclusion for subqueries with mergeappend
+-- should include 2 chunks
+:PREFIX SELECT device_id, time FROM (SELECT device_id, time FROM metrics_timestamptz WHERE time < '2000-01-10'::text::timestamptz ORDER BY device_id, time) m;
+                                                                QUERY PLAN                                                                 
+-------------------------------------------------------------------------------------------------------------------------------------------
+ Custom Scan (ConstraintAwareAppend) (actual rows=648 loops=1)
+   Hypertable: metrics_timestamptz
+   Chunks left after exclusion: 2
+   ->  Merge Append (actual rows=648 loops=1)
+         Sort Key: _hyper_5_17_chunk.device_id, _hyper_5_17_chunk."time"
+         ->  Index Only Scan using _hyper_5_17_chunk_metrics_timestamptz_device_id_time_idx on _hyper_5_17_chunk (actual rows=336 loops=1)
+               Index Cond: ("time" < ('2000-01-10'::cstring)::timestamp with time zone)
+               Heap Fetches: 336
+         ->  Index Only Scan using _hyper_5_18_chunk_metrics_timestamptz_device_id_time_idx on _hyper_5_18_chunk (actual rows=312 loops=1)
+               Index Cond: ("time" < ('2000-01-10'::cstring)::timestamp with time zone)
+               Heap Fetches: 312
+(11 rows)
+
 --generate the results into two different files
 \set ECHO errors
+--- Unoptimized results
++++ Optimized results
+@@ -1,6 +1,6 @@
+               setting              | value 
+ -----------------------------------+-------
+- timescaledb.disable_optimizations | on
++ timescaledb.disable_optimizations | off
+  timescaledb.enable_chunk_append   | on
+ (2 rows)
+ 
+--- Unoptimized results
++++ Optimized results
+@@ -1,7 +1,7 @@
+               setting              | value 
+ -----------------------------------+-------
+- timescaledb.disable_optimizations | on
+- timescaledb.enable_chunk_append   | on
++ timescaledb.disable_optimizations | off
++ timescaledb.enable_chunk_append   | off
+ (2 rows)
+ 
+  time | temp | colorid 
```

Comment:
what is this line doing here?

Reply:
that is the context from the unified diff, 3 lines before and 3 lines after changed lines